@stellar/design-system-website/src/components/Details/styles.scss

```diff
@@ -1,28 +1,76 @@
 .Details {
-  &__example__grid {
-    display: grid;
-    grid-template-columns: repeat(auto-fill, minmax(9rem, 1fr));
-    gap: 1rem;
+  &__example {
+    &__container {
+      display: block;
+
+      &:not(:last-child) {
+        margin-bottom: 1rem;
+      }
 
-    .Details__example__container {
+      @media (min-width: 900px) {
```

Comment:
should 900px be a var?

Reply:
It would be cool if we could use variables in CSS media queries. [At the moment it's not supported](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries).